web_src/js/features/projects.js

```diff
@@ -37,26 +37,29 @@ export default async function initProject() {
     },
   );
 
+  const moveIssue = (e) => {
+    $.ajax(`${e.to.dataset.url}/${e.item.dataset.issue}/${e.newIndex}`, {
```

Comment:
TODO: Works in general, but `e.newIndex` is `0` if I move it to the top, if this is repeated for other project-issuses as well the sorting id of all of them is `0` and sorting breaks. For sorting the colums a for-loop is being used to get the correct position, but that looks rather hacky to me 🙈

Reply:
I think all other `sorting>newIndex` rows should be update as `sorting = sorting+1`, if I understand correctly...

Reply:
Should be fine now.